packages/model/addon/-private/belongs-to.js

```diff
@@ -149,7 +149,7 @@ function belongsTo(modelName, options) {
         }
         if (opts.hasOwnProperty('serialize')) {
           warn(
-            `You provided a serialize option on the "${key}" property in the "${this._internalModel.modelName}" class, this belongs in the serializer. See Serializer and it's implementations https://emberjs.com/ember-data/release/classes/Serializer`,
+            `You provided a serialize option on the "${key}" property in the "${this._internalModel.modelName}" class, this belongs in the serializer. See Serializer and it's implementations /ember-data/release/classes/Serializer`,
```

Comment:
since this is a warning in the console I don't think we can use a relative link. We should update it to point at the serializer package overview though :)

Reply:
I fixed the link, but I think pointing it to the serializer package overview should be left to a separate PR, as that is not really fixing docs.